### DISCUSSION TITLE
Bulk editing products and offers

### DISCUSSION THREAD

**Discussion by lautsevich** [lovata/oc-shopaholic-plugin#348] [Ideas]
Bulk editing of catalog items is frequently asked functionality. By opening this issue we're starting gathering of the functional requirement. Feel free to share your ideas!

P.S. Please, vote for this feature as well:
- 👍️ — I need this in a current project.
- ❤️ — I'll need this in the future project.
- 👀️ — I needed this in a recent project.

**Comment by FlusherDock1** [comment]
The smaller shops that have around 10k products and same offers really need some tool, that can make content manager life better. The main headace comes from filling in properties for products, that's very common that many of them share same characteristics across thousands of similiar products. We hope that you'll add some interface, that can fully close this problem.

**Comment by lautsevich** [comment]
@FlusherDock1, thx for your opinion. Di you have ideas regarding the possible user scenario to do this? Or maybe about the UI?

**Comment by FlusherDock1** [comment]
@lautsevich У нас была идея разработать сторонний интерфейс, который интегрируется с сайтом на шопоголике и дает возможность управлять каталогом, заказами и настройками через него, имеет возможность интегрировать сразу несколько магазинов, и управлять всеми ими через один софт. Для контент-менеджера в нашей студии было-бы удобно так управлять каталогом существующего магазина и всех последующих.

Такой вариант дает нам возможность реализовывать любые дизайнерские изыски, написать интерфейс на любом удобном для нас стеке и реализовать любые возможные формы для массового редактирования.

А интерфейс админки шопоголика остается неизменным, и новый интерфейс, который должен быть сложнее функциональнее не будет мешать тем, кому это не надо.

**Comment by lautsevich** [comment]
@FlusherDock1, it could be a good alternative for complex retail businesses! Lrt's do it! And don't forget to share it to the community! =)

As for the built-in functionality, do you have ideas?
